---a/net/minecraft/block/BlockPackedIce.java
+++b/net/minecraft/block/BlockPackedIce.java
@@ -1,21 +1,21 @@
 package net.minecraft.block;
 
 import java.util.Random;
+
 import net.minecraft.block.material.Material;
 import net.minecraft.creativetab.CreativeTabs;
 
-public class BlockPackedIce extends Block
-{
+public class BlockPackedIce extends Block {
+
     private static final String __OBFID = "CL_00000283";
 
-    public BlockPackedIce()
-    {
+    public BlockPackedIce() {
         super(Material.packedIce);
         this.slipperiness = 0.98F;
         this.setCreativeTab(CreativeTabs.tabBlock);
     }
 
-    public int quantityDropped(Random random)
-    {
+    public int quantityDropped(Random random) {
         return 0;
     }
-}
+}
